base/300-deployment.yaml

```diff
@@ -5,18 +5,31 @@ metadata:
   name: tekton-dashboard
   namespace: tekton-pipelines
   labels:
+    app.kubernetes.io/name: dashboard
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/part-of: tekton-dashboard
+    dashboard.tekton.dev/release: "devel"
     app: tekton-dashboard
     version: "devel"
-    dashboard.tekton.dev/release: "devel"
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: tekton-dashboard
+      app.kubernetes.io/name: dashboard
+      app.kubernetes.io/component: dashboard
+      app.kubernetes.io/instance: default
+      app.kubernetes.io/part-of: tekton-dashboard
   template:
     metadata:
       name: tekton-dashboard
       labels:
+        app.kubernetes.io/name: dashboard
+        app.kubernetes.io/component: dashboard
+        app.kubernetes.io/instance: default
```

Comment:
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#applications-and-instances-of-applications

Why `default` for instance and not dashboard? All other changes look good

Reply:
The `app.kubernetes.io/instance` label shouldn't express the application itself but the instance of the application, i feel like these manifests express the `default` deployment of the `dashboard` application.

Also it's more consistent, i did the same thing for `pipelines` and `triggers` (setting `app.kubernetes.io/instance` to `default`). I guess it will be easier to remember when trying to lookup a resource.

Reply:
For example, when deploying with helm i set the value to the release name : https://github.com/tektoncd/experimental/pull/528/files#diff-718e9764773438f64515998a5c46e5a1R50

Reply:
Got it, thanks for the clarifications